tests/system/providers/amazon/aws/example_emr.py

```diff
@@ -124,3 +127,15 @@
         step_checker,
         cluster_remover,
     )
+
+    from tests.system.utils.watcher import watcher
+
+    # This test needs watcher in order to properly mark success/failure
+    # when "tearDown" task with trigger rule is part of the DAG
+    list(dag.tasks) >> watcher()
```

Comment:
Which task is the teardown task in this DAG? Should the "remove_cluster" task have a `trigger_rule` of `ALL_DONE` (assuming this is the teardown task).

If there isn't a teardown task then [the `watcher()` isn't needed](https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-47+New+design+of+Airflow+System+Tests#AIP47NewdesignofAirflowSystemTests-Processofmigrationindetails:~:text=If%20you%20have%20a%20teardown%20task%20(with%20TriggerRule.ALL_DONE)%2C%20you%20need%20to%20add%20the%20watcher%20task%20dependency%20for%20all%20other%20tasks%20by%20adding%20these%20lines%20of%20code%20below%20all%20other%20task%20dependencies%3A).